assets/source/setup-guide/app/views/WizardApp.test.js

```diff
@@ -0,0 +1,188 @@
+jest.mock( '@woocommerce/tracks' );
+jest.mock( '../data/settings/selectors', () => ( {
+	...jest.requireActual( '../data/settings/selectors' ), // import and retain the original functionalities
+	isTrackingConfigured: jest.fn().mockReturnValue( false ),
+	isDomainVerified: jest.fn().mockReturnValue( false ),
+} ) );
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+import '@testing-library/jest-dom';
+import { fireEvent, render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import WizardApp from './WizardApp';
+import '../../../tests/custom-matchers';
+import {
+	isDomainVerified,
+	isTrackingConfigured,
+} from '../data/settings/selectors';
+
+recordEvent.mockName( 'recordEvent' );
+
+//Needed to be able to render the Stepper component
+jest.mock( '../steps/SetupAccount', () => () => null );
+jest.mock( '../steps/ClaimWebsite', () => () => null );
+jest.mock( '../steps/SetupTracking', () => () => null );
+
+const stepOne = /Set up your business account/;
+const stepTwo = /Claim your website/;
+const stepThree = /Track conversions/;
+
+afterEach( () => {
+	jest.clearAllMocks();
```

Comment:
❔  I am curious about why this is needed? 🤔

Reply:
Thanks for pointing this out. I have double-checked and I see that is unnecessary as we are not really asserting any `fn.mock.calls` or similar.  

See [801bf25](https://github.com/woocommerce/pinterest-for-woocommerce/pull/318/commits/801bf25bb200d35f42c44583e1a4ff5a559bb318)